src/router/index.ts

```diff
@@ -25,18 +28,42 @@ const router = createRouter({
       path: "/",
       name: "home",
       component: HomeView,
+      meta: { showNav: true, showNotitie: true, showSearch: true },
```

Comment:
Could we type this? I can now say (e.g.) `meta: { showSomethingThatDoesntExist: true }`.

Also, right now, we're explicitly telling everything if we want to show or hide them. Maybe we could swap it around to only explicitly having to say we want to hide elements?

Reply:
mee eens, maar misschien even afwachten hoe het menu zich verder ontwikkelt? heb het idee dat dit nog wel wat zal veranderen